src/studio/src/designer/frontend/app-development/features/administration/components/Administration.tsx

```diff
@@ -267,8 +281,9 @@ const mapStateToProps = (
 ): IAdministrationComponentProps => {
   return {
     classes: props.classes,
+    dispatch: props.dispatch,
     initialCommit: state.serviceInformation.initialCommit,
-    language: state.language,
+    language: state.language.language,
```

Comment:
Have we added another layer here? Could we differentiate these objects? `state.languageState.language` or something similiar?

Reply:
We haven't added another layer, I have no idea how this worked before 😱 Most of the other components use the `state.language.language` syntax.  But I agree, we could rename the state portion so it's more obviously the languageState.